packages/react-components/src/hooks/useAlarms/hookHelpers/predicates.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  AlarmAssetModelRequest,
+  AlarmAssetRequest,
+  AlarmCompositeModelRequest,
+  AlarmInputPropertyRequest,
+  AlarmRequest,
+} from '../types';
```

Comment:
Nit: Use type imports when importing types. Same comment applies for the entire change.